Source/WebCore/html/TextFieldInputType.cpp

```diff
@@ -209,7 +209,7 @@ auto TextFieldInputType::handleKeydownEvent(KeyboardEvent& event) -> ShouldCallB
         return ShouldCallBaseEventHandler::Yes;
 #if ENABLE(DATALIST_ELEMENT)
     const String& key = event.keyIdentifier();
-    if (RefPtr suggestionPicker = m_suggestionPicker; suggestionPicker && (key == "Enter"_s || key == "Up"_s || key == "Down"_s)) {
+    if (RefPtr suggestionPicker = m_suggestionPicker; suggestionPicker && (key == "Enter"_s || key == "Up"_s || key == "Down"_s || key == "U+001B"_s)) {
```

Comment:
Can you add a comment saying which key this is

Reply:
hmm we have this in quite a few places with no comment

Reply:
I think this might better be should be translated to "Escape" in `PlatformEventFactory*`

Reply:
I filed https://bugs.webkit.org/show_bug.cgi?id=286147 for that.